package.json

```diff
@@ -81,15 +81,18 @@
         "test:e2e:docker": "npm-run-all --serial test:e2e:docker:build \"test:e2e:docker:run {@}\" --",
         "test:unified": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest --projects src/tests/electron --runInBand --forceExit --detectOpenHandles",
         "test:report:e2e": "lerna --scope accessibility-insights-report-e2e-tests run test",
+        "tbuild": "yarn build && yarn type-check",
```

Comment:
It would be good to update `building-web.md` with explanations of when to use `tbuild` vs `build`